### ISSUE TITLE
Deal with name collisions in classes

### ISSUE THREAD

**Issue by asbjornh** [Creuna-Oslo/prop-types-csharp-webpack-plugin#24]
```
Component: {
  component: PropTypes.string
};
```
Will result in an invalid class because C# classes can't have properties of the same name as the class name. Either throw an error or solve this another way. Prepend the class name maybe

**Comment by pellebjerkestrand** [comment]
It can, but it can't have the same name for more than one member.

This is fine:
```c#
public class Example {
  public string Example { get; set; }
}
```

This is not fine:
```c#
public class Example {
  public string Example { get; set; } // <- Member one

  public class Example {} // <- Member two
}
```

**Comment by asbjornh** [comment]
@pellebjerkestrand  It turns out that this isn't true!

**Comment by asbjornh** [comment]
Fixed in `0.1.6`
